changelog-2022.07.01-6.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/qt/res/app.organicmaps.desktop.metainfo.xml b/qt/res/app.organicmaps.desktop.metainfo.xml
+index a8df72cd76..4a1802dbda 100644
+--- a/qt/res/app.organicmaps.desktop.metainfo.xml
++++ b/qt/res/app.organicmaps.desktop.metainfo.xml
+@@ -107,6 +107,21 @@
+   </screenshots>
+ 
+   <releases>
++   <release version="2022.07.01-6" date="2022-07-01">
++     <description>
++       <ul>
++         <li>Improved/fixed the drawing of pattern lines</li>
++         <li>Icons for information=board/map</li>
++	  <li>Fixed categories search with stop words</li>
```

Comment:
Indentation is not beautiful.

Reply:
Could you be more specific? I don't see any difference compared to the original `app.organicmaps.desktop.metainfo.xml` or the previous patch `changelog-2022.06.18-2.patch`.

Reply:
I assume you mixed tabs and spaces.

![image](https://user-images.githubusercontent.com/170263/178131369-5fe84210-a070-47ad-bb09-caae576256bd.jpeg)

Reply:
Ohhh. That should be fixed in the latest commit.
Do we want to handle these kinds of issues automatically?
For example in gitconfig or a GH workflow or something similar?

Reply:
Hi @biodranik is there anything else needed before merging this PR?